libs/blocks/bulk/bulk.js

```diff
@@ -67,117 +67,117 @@ function prettyDate() {
-}) {
+function StatusTitle({ bulkTriggered, submittedAction, urlNumber }) {
   const name = getActionName(submittedAction, true);
   const URLS = (urlNumber > 1) ? 'URLS' : 'URL';
   return bulkTriggered && html`
-      <div class="bulk-status-head">
-          <div class="bulk-status-head-title">
-              STATUS ${name} ${urlNumber} ${URLS}
-          </div>
-          <div>
-              <button class="bulk-status-head-copy" onclick=${copyResults} title="Copy results"></button>
-              <button class="bulk-status-head-toggle-urls" ref="${bulkInputToggleElt}" onclick=${showHideUrls} title="Show/hide urls"></button>
-          </div>
-      </div>
+    <div class="bulk-status-head">
+      <ul class="bulk-status-head-title">
+        <li><span>STATUS:</span> ${name} ${urlNumber} ${URLS}</li>
+      </ul>
+    </div>
+  `;
+}
+
+function StatusRow({ row }) {
+  const timeStamp = prettyDate();
+  const errorStyle = 'status-error';
+  const previewStatusError = row.status.preview !== 200
+    ? errorStyle
+    : '';
+  const publishStatusError = row.status.publish !== 200
+    ? errorStyle
+    : '';
+
+  return html`
+    <tr class="bulk-status-row">
+      <td class="bulk-status-url"><a href="${row.url}" target="_blank">${row.url}</a></td>
+      <td class="bulk-status-preview ${previewStatusError}">
+        ${row.status.preview === 200 && timeStamp} ${bulkPreviewStatus(row)}
+      </td>
+      <td class="bulk-status-publish ${publishStatusError}">
+        ${row.status.publish === 200 && timeStamp} ${bulkPublishStatus(row)}
+      </td>
+    </tr>
   `;
 }
 
-function StatusContent({ resultsElt, result }) {
+function StatusContent({ resultsElt, result, submittedAction }) {
+  const name = getActionName(submittedAction);
+  const displayClass = 'did-bulk';
+  const bulkPreviewed = name.toLowerCase().includes('preview') ? displayClass : '';
+  const bulkPublished = name.toLowerCase().includes('publish') ? displayClass : '';
   return html`
     <table class="bulk-status-content" ref="${resultsElt}">
-        ${result && html`
-            <colgroup>
-                <col class="bulk-status-content-col1"/>
-                <col class="bulk-status-content-col2"/>
-                <col class="bulk-status-content-col3"/>
-                <col class="bulk-status-content-col4"/>
-            </colgroup>
-            <tr class="bulk-status-header-row">
-                <th class="bulk-status-header-url">Preview</th>
-                <th>Status</th>
-                <th class="bulk-status-header-url">Publish</th>
-                <th>Status</th>
-            </tr>
-            ${result.map((row) => html`<${StatusRow} row=${row} />`)}
-        `}
+      ${result && html`
+        <tr class="bulk-status-header-row">
+          <th>URL</th>
+          <th class="${bulkPreviewed}">Previewed</th>
+          <th class="${bulkPublished}">Published</th>
+        </tr>
+        ${result.reverse().map((row) => html`<${StatusRow} row=${row} />`)}
```

Comment:
Why are the results reversed? Shouldn't they be displayed in the same order as the input?